src/Orchard.Cms.Web/Modules/Orchard.Demo/Shapes/DemoShapeProvider.cs

```diff
@@ -14,12 +14,15 @@ public void Discover(ShapeTableBuilder builder)
         }
 
         [Shape]
-        public IHtmlContent Baz(string text, int count)
+        public IHtmlContent Baz(string Text, int Count)
```

Comment:
These properties should work in lower case too, if you have to upper case them not just for style, then there is a problem. When the attribute binding is resolved, it will look for arguments in a case insensitive way.